test_data/gbif/test_data.sql

```diff
@@ -277,7 +277,7 @@ CREATE MATERIALIZED VIEW family_stats AS (
         family AS name, -- noqa: RF04
         count(*) AS count,
         tsrange('[' || min(eventdate) || ',' || max(eventdate) || ']')
-        AS "time",
+        AS time, -- noqa: RF04
```

Comment:
why remove the quote if it triggers a lint?

Reply:
War ein Autofix vom Linter. Dem ist egal, ob time oder "time".